clientv3/concurrency/election.go

```diff
@@ -55,8 +55,21 @@ func ResumeElection(s *Session, pfx string, leaderKey string, leaderRev int64) *
 	}
 }
 
-// Campaign puts a value as eligible for the election. It blocks until
-// it is elected, an error occurs, or the context is cancelled.
+// Campaign puts a value as eligible for the election on the prefix
+// key.
+// Multiple sessions can participate in the election for the
+// same prefix, but only one can be the leader at a time.
+// A session succeeds in election if it can create a key with the
```

Comment:
62-66 are implementation details that we want to hide from the users. simply telling users what to expect is good enough.